--- old-configure.orig.in	2019-03-02 21:49:16.000000000 +0100
+++ old-configure.in	2019-03-04 12:14:23.842685662 +0100
@@ -1715,7 +1715,7 @@
     MOZ_SYSTEM_LIBEVENT=
 elif test -z "$MOZ_SYSTEM_LIBEVENT"; then
     CFLAGS="-I${LIBEVENT_DIR}/include $CFLAGS"
-    LDFLAGS="-L${LIBEVENT_DIR}/lib $LDFLAGS"
+    LDFLAGS="-L${LIBDIR} $LDFLAGS"
     MOZ_CHECK_HEADER(event.h,
         [if test ! -f "${LIBEVENT_DIR}/include/event.h"; then
              AC_MSG_ERROR([event.h found, but is not in ${LIBEVENT_DIR}/include])
@@ -1724,7 +1724,7 @@
     AC_CHECK_LIB(event, event_init,
                  [MOZ_SYSTEM_LIBEVENT=1
                   MOZ_LIBEVENT_CFLAGS="-I${LIBEVENT_DIR}/include"
-                  MOZ_LIBEVENT_LIBS="-L${LIBEVENT_DIR}/lib -levent"],
+                  MOZ_LIBEVENT_LIBS="-L${LIBDIR} -levent"],
                  [MOZ_SYSTEM_LIBEVENT= MOZ_LIBEVENT_CFLAGS= MOZ_LIBEVENT_LIBS=])
 fi
 CFLAGS=$_SAVE_CFLAGS
